www/js/components/add-point-page.js

```diff
@@ -11,25 +14,35 @@ import { connect } from 'react-redux';
 // import leaflet components
 import { Marker, Map, TileLayer } from 'react-leaflet';
 
+const styleShadow = {marginLeft: '4px', marginTop: '18px', width: '25px', height: '41px'}
+const styleMarker = {marginLeft: '0px', marginTop: '18px', height: '41px'}
+
 export class AddPointPage extends Component {
   constructor(props) {
     super(props);
-    this.state = {startCenter: [0,0], center: {lat:0, lng:0}};
+    this.state = {center:[0,0]}
   }
 
-  onMapMoved(e) {
-    this.setState({center: e.target.getCenter()});
+  updateCenter(leafletMap){
```

Comment:
This callback shouldn't be given the whole `leafletMap` instance. It should just be passed `center`.